pkgs/applications/networking/browsers/floorp/default.nix

```diff
@@ -7,17 +7,19 @@
 
 ((buildMozillaMach rec {
   pname = "floorp";
-  packageVersion = "11.5.1";
+  packageVersion = "11.6.0";
   applicationName = "Floorp";
   binaryName = "floorp";
-  version = "155.4.0";
+
+  # Must match the contents of `browser/config/version.txt` in the source tree
+  version = "155.5.0";
```

Comment:
I don't understand how these version numbers relate to Mozilla's versioning. Firefox mainline is at 120.0 right now, and ESR at 115.5.0.

The `buildMozillaMach` function uses `version` to conditionally applies patches. That's probably going to break things some day.

Reply:
Well, Floorp builds from the ESR 115.x branch of Firefox, that is basically why.

But you're right, it could potentially break - I'll take a more in-depth look at what is being applied and how that (could) affect builds, before anything really does.

Thing is, completely de-coupling this from the Mozilla version line would probably mean teaching `buildMozillaMach` how to deal with custom version numbers and just make things more complicated/intricate.

Thanks for taking a look!